setup.py

```diff
@@ -71,6 +71,7 @@ def has_environment_marker_platform_impl_support():
         'Twisted>=17.9.0',
         'cryptography>=2.0',
         'cssselect>=0.9.1',
+        'itemloaders>=1.0.0',
```

Comment:
Could you please check that itemloader's setup.py uses the same minimal package requirements as Scrapy?

Reply:
What do you mean by "the same minimal package requirements as Scrapy"?

Reply:
If itemloaders require e.g. parsel 1.6, and scrapy requires 1.5, and now scrapy requires itemloaders, than we're implicitly bumping minimal supported parsel version in scrapy.

Reply:
I'll update the requirements there..
Though, `jmespath` was optional in scrapy, but is a requirement in `itemloaders`
It shouldn't be a problem, but it changed.

Reply:
https://github.com/scrapy/itemloaders/pull/10